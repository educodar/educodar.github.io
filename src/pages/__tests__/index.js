import React from "react"
import renderer from "react-test-renderer"

import HomePage from ".."

describe("Home page", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HomePage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
