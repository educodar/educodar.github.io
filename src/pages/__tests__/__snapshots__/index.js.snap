// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
<article
  className="MuiBox-root MuiBox-root-2"
>
  <div
    className="MuiBox-root MuiBox-root-3"
  >
    <div
      className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        className="MuiCardMedia-root"
      />
      <div
        className="MuiCardContent-root"
      >
        <h2
          className="MuiTypography-root MuiTypography-h2 makeStyles-h2-4 MuiTypography-gutterBottom"
        >
          Aprenda programação gratuitamente.
        </h2>
        <a
          aria-disabled={false}
          className="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiTypography-colorPrimary"
          href="/quero-aprender"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
          target="_self"
        >
          <span
            className="MuiButton-label"
          >
            Quero aprender
          </span>
        </a>
         
        <a
          aria-disabled={false}
          className="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiButtonBase-root MuiButton-root MuiButton-contained MuiTypography-colorPrimary"
          href="/tire-suas-duvidas"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
          target="_self"
        >
          <span
            className="MuiButton-label"
          >
            Tire suas dúvidas
          </span>
        </a>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
    >
      <section
        className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <h2
            className="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom"
          >
            Para quem é o Educodar?
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
          >
            Tem pelo menos 14 anos e estuda em uma escola pública de Belo Horizonte (
            <abbr
              title="Minas Gerais"
            >
              MG
            </abbr>
            )
             
            <wbr />
            ou Porto Alegre (
            <abbr
              title="Rio Grande do Sul"
            >
              RS
            </abbr>
            )?
          </p>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
          >
            Então você pode se
             
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="/quero-aprender"
              onBlur={[Function]}
              onFocus={[Function]}
              target="_self"
            >
              inscrever para estudar no 
              <strong>
                Educodar
              </strong>
            </a>
            .
          </p>
        </div>
      </section>
    </div>
    <hr
      className="MuiDivider-root"
    />
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
    >
      <section
        className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          className="MuiCardContent-root"
        >
          <h2
            className="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom"
          >
            Divulgue o Educodar na sua escola
          </h2>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
          >
            Você estuda ou leciona em uma escola em Belo Horizonte (
            <abbr
              title="Minas Gerais"
            >
              MG
            </abbr>
            )? Ajude-nos a divulgar.
          </p>
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
          >
            Leve mais
             
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="/tire-suas-duvidas"
              onBlur={[Function]}
              onFocus={[Function]}
              target="_self"
            >
              informações sobre o Educodar
            </a>
             
            para sua escola.
          </p>
        </div>
      </section>
    </div>
  </div>
</article>
`;
